src/components/Calculator.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+import './Calculator.css';
+
+export default class Calculator extends Component {
+  render() {
+    return (
+      <div className="main-container">
+        <div className="result-display" />
+        <button type="button" onClick={this.btnOnClickOperator} className="btn-light btn-zero">0</button>
+        <button type="button" onClick={this.btnOnClickOperator} className="btn-light btn-dot">.</button>
```

Comment:
- [ ] Great effort working on your Math-magician project, As stated in the project requirement it is paramount we make use of the react `functional component` instead of class component.
E.g 👇 

```JS
import React from "react"

function Calculator() {
  return <div>{/* Calculator app design goes here */}</div>
}

export default Calculator
```